src/Xsc/XscEventClient.ts

```diff
@@ -0,0 +1,89 @@
+import { HttpClient } from '..';
+import { IRequestParams } from '../HttpClient';
+
+import {
+    IClientResponse,
+    ILogger,
+    ScanEventStatus,
+    ScanEventType,
+    XscLog,
+    StartScanRequest,
+    ScanEvent,
+} from '../../model/';
+
+export class XscEventClient {
+    static readonly eventEndpoint: string = 'api/v1/event';
```

Comment:
NP with this one, just where does the "/xsc/" come from?

Reply:
this is only the partial end point. they can still change the basic url